api/src/main/java/mcoffin/rogue/EncoderControl.java

```diff
@@ -0,0 +1,4 @@
+public interface EncoderDrive {
+  public double getEncoderDistance();
```

Comment:
Think about this for a second. You might not be telling it to drive forward. This should really be `getEncoderPosition`, and it should return a vector representing the robot's position relative to the last encoder reset.